composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "anttiviljami/wp-libre-form",
+  "name": "devgeniem/wp-libre-form",
```

Comment:
You don't have to, I just had to quickly make this fork available to install via Composer and this was the fastest way to do so. There probably is better, but I'll change it back sooner or later. :)